test/specs/landingpage.js

```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+module.exports = { 
+
+    'Testing Pagevamp logo' : function (browser) {
+        browser
+        .url("https://www.pagevamp.com/")
+        .waitForElementVisible("#topmost-nav a.pv-logo",1000)
+        .click("#topmost-nav a.pv-logo")
+        .url(function(result) {
+                this.assert.equal(result.value, "https://www.pagevamp.com/");  //clicking Pagevamp logo
+
+            })
+        .pause(3000)
+        .end();
+    },
+
+    'Testing if "How it works" is working' : function (browser) {
+        browser
+        .url("https://www.pagevamp.com/")
+        .click(".navbar-right li:nth-child(1) a")
+        .url(function(result) {
+                this.assert.equal(result.value, "https://www.pagevamp.com/#how-it-works");   //scroll to How it work section
+                this.assert.containsText("#how-it-works-section","Website in seconds");     //testing "Website in seconds" text
```

Comment:
Why do we need this?

Reply:
added command for "Watch it in action" youtube video link